static/app/components/events/searchBar.tsx

```diff
@@ -33,63 +33,60 @@ const SEARCH_SPECIAL_CHARS_REGEXP = new RegExp(
   'g'
 );
 
-const getFunctionTags = (fields: Readonly<Field[]>) =>
-  Object.fromEntries(
```

Comment:
I'm not sure, but I would expect fromEntries to execute another On loop which we can avoid entirely by just doing filtering inside .reduce and return the acc. This is the majority of the changes